machine-learning/solutions/kubeflow-pipelines-sentiment-analysis/sentiment-analysis/src/main/java/SentimentAnalysis.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
```

Comment:
My suggestion would be to arrange this code into some packages. Something like this:

com.google.cloud.pso.pipelines  // The main Dataflow pipeline
com.google.cloud.pso.common  // For common classes for e.g. Doc